sympy/simplify/tests/test_gammasimp.py

```diff
@@ -67,8 +67,8 @@ def test_gammasimp():
     assert gammasimp(e**2/gamma(k + 1)) == k/gamma(k)
     a = R(1, 2) + R(1, 3)
     b = a + R(1, 3)
```

Comment:
This was an assertion where the left and rhs of the `==` were disconnected; so searching for more than `==` expressions caught this.